data_collection/gazette/spiders/rs_lajeado.py

```diff
@@ -0,0 +1,63 @@
+from datetime import date
+from urllib.parse import urlencode
+
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RsLajeadoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4311403"
+    BASE_URL = "https://lajeado.rs.gov.br"
+
+    name = "rs_lajeado"
+    start_date = date(2016, 4, 5)
+    params = {
+        "titulo": "Diário Oficial",
+        "template": "conteudo",
+        "codigoCategoria": 1012,
+    }
+
+    def start_requests(self):
+        params = urlencode(self.params)
+        yield Request(self.BASE_URL + "/?" + params)
+
+    def parse(self, response):
+        pdfs = response.css("ul.anexos li a")
+        for item in pdfs:
+            filename = item.css("::text").get()
+
+            kwargs = {"filedate": self.extract_date(filename)}
+
+            href = item.attrib["href"]
+            href = response.urljoin(href)
+
+            yield Request(href, method="GET", callback=self.parse_url, cb_kwargs=kwargs)
+
+    def parse_url(self, response, **kwargs: dict):
+        url = response.css("div.topoAbreAnexoDownload a::attr(href)").get()
+        yield Gazette(
+            date=kwargs["filedate"],
+            file_urls=[url],
+            power="legislative",
+            territory_id=self.TERRITORY_ID,
+        )
+
+    @staticmethod
+    def extract_date(filename: str) -> date:
```

Comment:
**suggestion**: We don't use type hints in this project. It would be great to keep this spider following the same pattern.